ccdproc/combiner.py

```diff
@@ -380,111 +383,108 @@ def average_combine(self, scale_func=ma.average, scale_to=None, uncertainty_func
         return combined_image
 
 
-def combine(img_list, output_file=None, method='average', weights=None, scale=None, mem_limit=16e9,
+def combine(img_list, output_file=None, method='average', weights=None,
+            scale=None, mem_limit=16e9,
             minmax_clip=False, minmax_clip_min=None, minmax_clip_max=None,
-            sigma_clip=False, sigma_clip_low_thresh=3, sigma_clip_high_thresh=3,
+            sigma_clip=False,
+            sigma_clip_low_thresh=3, sigma_clip_high_thresh=3,
             sigma_clip_func=ma.mean, sigma_clip_dev_func=ma.std, **ccdkwargs):
-
-    """Convenience function for combining multiple images
+    """
+    Convenience function for combining multiple images.
 
     Parameters
     -----------
-    img_list : `list`, 'string'
-        A list of fits filenames or CCDData objects that will be combined together.
-        Or a string of fits filenames seperated by comma ",".
+    img_list : list or string
+        A list of fits filenames or CCDData objects that will be combined
+        together. Or a string of fits filenames seperated by comma ",".
 
-    output_file: 'string', optional
-        Optional output fits filename to which the final output can be directly written.
+    output_file : string or None, optional
+        Optional output fits filename to which the final output can be directly
+        written.
+        Default is ``None``.
 
-    method: 'string' (default average)
-        Method to combine images.
-             'average' : To combine by calculating average
-             'median'  : To combine by calculating median
+    method : string
+        Method to combine images:
 
-    weights: `~numpy.ndarray`, optional
+        - ``'average'`` : To combine by calculating the average.
+        - ``'median'`` : To combine by calculating the median.
+
+        Default is ``'average'``.
+
+    weights : `numpy.ndarray` or None, optional
         Weights to be used when combining images.
         An array with the weight values. The dimensions should match the
         the dimensions of the data arrays being combined.
+        Default is ``None``.
 
-    scale : function or array-like or None, optional
+    scale : function or `numpy.ndarray`-like or None, optional
         Scaling factor to be used when combining images.
         Images are multiplied by scaling prior to combining them. Scaling
         may be either a function, which will be applied to each image
         to determine the scaling factor, or a list or array whose length
         is the number of images in the `Combiner`. Default is ``None``.
 
-    mem_limit : float (default 16e9)
+    mem_limit : float, optional (default 16e9)
```

Comment:
have the default value in the detailed description (as you do it everywhere else)